.tool-versions

```diff
@@ -1 +1 @@
-ruby 2.7.1
```

Comment:
looks like you need to update your local ruby? `asdf local ruby 2.7.1` ?

Reply:
Thanks!